### PR TITLE
Add addon PHONY

### PR DESCRIPTION
**Pull request by sgnanase**
Adding kernel .deb for 4.19.80-celadon

Tracked-On:
Signed-off-by: sgnanase <sundar.gnanasekaran@intel.com>

### CODE REVIEW

**Review by zhy1030 — APPROVED** [review]

File: cic_dev/addon/setup-aic [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+./pre-requisites/setup-feature
+
+#Initiate Early-Launch
+./pre-requisites/launch-early
+
+#Remove un-used apks
+./pre-requisites/remove-app
+
+#Install Kernel
+./pre-requisites/setup-kernel
```
Comment (lwan89) [inline cic_dev/addon/setup-aic:16]: Why do we install host kernel by default?

File: cic_dev/addon/pre-requisites/launch-early [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+cat << 'EOF' | sudo tee /etc/rc.local 
+#!/bin/sh
+# start aic
+while (( $(ps -ef | grep -v grep | grep dockerd | wc -l) == 0 )); do sleep 1; done
+echo "PASSWD" | sudo -Sk /AIC_WORK_DIR/aic start
+exit 0
+EOF
+sudo chmod +x /etc/rc.local
+
+sudo sed -i "s|PASSWD|$password|g" /etc/rc.local
```
Comment (lwan89) [inline cic_dev/addon/pre-requisites/launch-early]: Save password in rc.local should have security issues. We'll merge a change soon which will allow start aic without root permission. Please remember to remove this later.

File: cic_dev/addon/setup-aic [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+#Install CIC
+cd $AIC_WORK_DIR
+echo "Installing CIC -- START"
+./aic install -e -d none
```
Comment (hongcheng-xie) [inline cic_dev/addon/setup-aic]: ./aic install  -u -e -d none
Pls add -u to  update docker image and remove unnecessary apks

File: cic_dev/addon/pre-requisites/launch-early (hunk 2 above)
Comment (lchen43) [reply cic_dev/addon/pre-requisites/launch-early]: save password is not good. must it be start before user login?

File: cic_dev/addon/pre-requisites/remove-app [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Email"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Gallery2"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Calendar"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/DeskClock"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Music"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Browser2"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/ExactCalculator"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/QuickSearchBox"] 
+EOF
+
```
Comment (hongcheng-xie) [inline cic_dev/addon/pre-requisites/remove-app:19]: Hi Sundar, 

According to Lenovo latest requirement, they want to keep following apps in the release image:
Settings, Gallery, Deskclock, Camera, Browser

Pls help to update remove apps content as follow in you PR_16.
#Add following content into update/Dockerfile
cat << EOF | sudo tee -a update/Dockerfile
RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/Contacts"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/MusicFX"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/DocumentsUI"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Email"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Calendar"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Music"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/ExactCalculator"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/QuickSearchBox"] 
EOF

File: cic_dev/addon/setup-aic (hunk 1 above)
Comment (sgnanase) [reply cic_dev/addon/setup-aic:16]: setup-aic is one time activity. Once done, we just have to use cfc/aic to start or stop container.
Also, We have dependency to use 4.19.80 kernel from chrome always.

File: cic_dev/addon/pre-requisites/launch-early (hunk 2 above)
Comment (sgnanase) [reply cic_dev/addon/pre-requisites/launch-early]: I have removed this step now.
Without sudo rc.local is executed fine. I verified on my local EB.
Please have a check.

File: cic_dev/addon/setup-aic (hunk 3 above)
Comment (sgnanase) [reply cic_dev/addon/setup-aic]: I have added this change as well... :)

File: cic_dev/addon/pre-requisites/remove-app (hunk 4 above)
Comment (sgnanase) [reply cic_dev/addon/pre-requisites/remove-app:19]: As discussed offline, except camera, everything else is taken care.

**Review by zhy1030 — APPROVED** [review]

**Pull request merged**